widget/label_test.go

```diff
@@ -18,24 +17,17 @@ func TestLabel_MinSize(t *testing.T) {
 	label.SetText("Longer")
 	assert.True(t, label.MinSize().Width > min.Width)
 }
-
-func TestLabel_Alignment(t *testing.T) {
```

Comment:
Is it right that this test was removed? (Always good to ask if a test is removed :) )

Reply:
Ooops don't know how it was removed... Added back.